### DISCUSSION TITLE
connection info for some clients not coming through

### DISCUSSION THREAD

**Discussion by yaroz** [rustdesk/rustdesk#8958] [General]
I'm migrating some servers from our old server to a new one that I just stood up.  About 4 out of 15 are not displaying user info.  I've tried uninstalling and reinstalling the client, but it's still not showing.  Not a show stopper by any means.. just wondering if anyone has any input, before I start rolling it out to a few hundred clients.

![image](https://github.com/user-attachments/assets/ca264aa5-508a-42f8-9b02-a2497d22a5e7)

**Comment by fufesou** [comment]
Hi, which peer tab is active?
If there is no logged in user on the target machine, the discovered peers do have this issue.

![image](https://github.com/user-attachments/assets/ae67631c-4572-4071-ae0e-d79e171b7ecb)

**Reply by yaroz** [reply]
doh.. that's it.  The computer was rebooted prior to the remote install and no user was logged in.
